staging/copy.sh

```diff
@@ -218,8 +222,11 @@ if [ "${VERIFYONLY}" = true ]; then
     fi
 fi
 
-echo "move to the client repo"
 # clean the ${CLIENT_REPO}
-ls "${CLIENT_REPO}" | { grep -v '_tmp' || true; } | xargs rm -rf
-mv "${CLIENT_REPO_TEMP}"/* "${CLIENT_REPO}"
+echo "move to the client repo"
+if [ "${DRY_RUN}" = false ]; then
```

Comment:
DRY_RUN only tests whether copy.sh works fine. This is for testing copy.sh itself. FAIL_ON_CHANGES checks whether copy.sh would do any changes.